src/signals/incident/components/form/WithHeading/WithHeading.tsx

```diff
@@ -0,0 +1,33 @@
+import type { FC } from 'react'
+import type { FormMeta, FormOptions } from 'types/reactive-form'
+
+import { Heading, themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+type WrapMeta = FormMeta & {
+  heading?: string
+  wrap?: FC
+}
+
+type WithHeadingProps = FormOptions & { meta: WrapMeta }
+
+const StyledHeading = styled(Heading)`
+  margin: ${themeSpacing(2, 0, 5)};
+`
+
+const WithHeading: FC<WithHeadingProps> = (props) => {
+  const { wrap, heading } = props.meta
+
```

Comment:
wrap refers to the wrapped component, right?
if so, this is a bit confusing.
I would prefer something like content, or input, or question...

Reply:
Yeah, I was debating a more descriptive name. Maybe `wrappedComponent`?

Reply:
👍